ui_framework/doc_site/src/components/guide_nav/guide_nav.js

```diff
@@ -8,75 +9,116 @@ import {
 
 import classNames from 'classnames';
 
-export const GuideNav = props => {
-  const classes = classNames('guideNav', {
-    'is-guide-nav-open': props.isNavOpen,
-  });
+export class GuideNav extends Component {
+  constructor() {
+    super();
 
-  const buttonClasses = classNames('guideNav__menu fa fa-bars', {
-    'is-menu-button-pinned': props.isNavOpen,
-  });
+    this.state = {
+      search: '',
+    };
 
-  const componentNavItems = props.components.map((item, index) => {
+    this.onSearchChange = this.onSearchChange.bind(this);
+  }
+
+  onSearchChange(event) {
```

Comment:
I tried it out and got a syntax error. I don't think it's valid ES6 class syntax. 🍂